ddtrace/bootstrap/sitecustomize.py

```diff
@@ -101,6 +103,10 @@ def update_patched_modules():
         log.debug('sitecustomize from user found in: %s', path)
         imp.load_module('sitecustomize', f, path, description)
 
-
+    # Loading status used in tests to detect if the `sitecustomize` has been
+    # properly loaded without exceptions. This must be the last action in the module
+    # when the execution ends with a success.
+    loaded = True
 except Exception as e:
+    loaded = False
```

Comment:
Because we catch any kind of exception, we should provide a loaded status since we "silently pass" to avoid crashing the application.